test/cli.js

```diff
@@ -42,4 +44,75 @@ describe('cli', function () {
       access(dest + 'bartest_0.0.1_amd64.deb', done)
     })
   })
+
+  describe('with duplicate dependencies', function (test) {
+    var dest = 'test/fixtures/out/tjaq/'
+    var config = 'test/fixtures/config.json'
+
+    // Default options (from src/installer.js)
+    var defaults = {
+      depends: ['gvfs-bin',
+        'libgconf2-4',
+        'libgtk2.0-0',
+        'libnotify4',
+        'libnss3',
+        'libxtst6',
+        'xdg-utils'],
+      recommends: ['pulseaudio | libasound2'],
+      suggests: ['gir1.2-gnomekeyring-1.0',
+        'libgnome-keyring0',
+        'lsb-release'],
+      enhances: [],
+      preDepends: []
```

Comment:
This is going to be very annoying to maintain. Can we make the default dependencies part of a module?

Reply:
This is a good idea, but it should go in another PR. This one is just to fix the bug I found using the command-line. Maybe we could move all the defaults to a module and not just the default dependencies.

Reply:
Filed https://github.com/unindented/electron-installer-debian/issues/116.